src/main/resources/view/MainWindow.fxml

```diff
@@ -32,28 +33,34 @@
             <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
           </Menu>
         </MenuBar>
+            <TabPane prefHeight="600.0" prefWidth="200.0" tabClosingPolicy="UNAVAILABLE">
+              <tabs>
+                <Tab fx:id="diaryTab" text="Diary" />
+                <Tab fx:id="modPlanTab" text="Mod plan" />
+                  <Tab fx:id="addBookTab" text="Add Book">
+                     <content>
 
-        <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
-          <padding>
-            <Insets top="5" right="10" bottom="5" left="10" />
-          </padding>
-        </StackPane>
+                       <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS" />
+                     </content>
+                  </Tab>
+                  <Tab fx:id="calenderTab" text="Calender" />
+                  <Tab fx:id="notesTab" text="Notes" />
+                  <Tab fx:id="profileTab" text="Profile" />
+              </tabs>
+            </TabPane>
+            <StackPane fx:id="resultDisplayPlaceholder" maxHeight="100" styleClass="pane-with-border">
+               <padding>
+                  <Insets bottom="5" left="10" right="10" top="5" />
+               </padding>
+            </StackPane>
```

Comment:
maybe can put the tabs in the same line and same styling as the "File" and "Help" above.